spring-cloud-kubernetes-leader/pom.xml

```diff
@@ -37,16 +37,31 @@
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-actuator</artifactId>
+			<optional>true</optional>
+		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-webflux</artifactId>
+			<scope>test</scope>
+		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
```

Comment:
Why do we need to `actuator-starter` and `actuator`?

Reply:
yup, just use `starter-actuator` and make it optional.

Reply:
This usage of optional `actuator` for the main code and `starter-actuator` for testing is the same approach as both the `core` and `config` `pom.xml`.

I assumed I should be consistent? My understanding was that `starter` is only required for the tests to auto-detect etc.

Reply:
it's what we've done in other places.